UDF native example compile failed: too few arguments in function call

**Describe the bug**
first seen in examples-udf-examples-native run: 458

`error: too few arguments in function call` which is related to the recent cudf update at https://github.com/rapidsai/cudf/pull/20258

mismatched `cuco@0.0.1 (d36905c69ce02d74abdd31dc864ce3e1ffc5a7db)` commit and cudf changes (requires the latest commit https://github.com/NVIDIA/cuCollections/commits/dev/) which is from https://github.com/rapidsai/rapids-cmake/pull/929/files

```
[2025-10-22T07:54:37.807Z] [INFO]      [exec] /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/storage/bucket_storage.inl(103): error: too few arguments in function call

[2025-10-22T07:54:37.807Z] [INFO]      [exec]       slots_{allocator_.allocate(capacity()), slot_deleter_type{capacity(), allocator_}}

[2025-10-22T07:54:37.807Z] [INFO]      [exec]                                            ^

[2025-10-22T07:54:37.807Z] [INFO]      [exec]           detected during:

[2025-10-22T07:54:37.807Z] [INFO]      [exec]             instantiation of "cuco::bucket_storage<T, BucketSize, Extent, Allocator>::bucket_storage(Extent, const Allocator &) [with T=cudf::size_type, BucketSize=1, Extent=cuco::valid_extent<int64_t, 18446744073709551615UL>, Allocator=rmm::mr::polymorphic_allocator<char>]" at line 58 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/storage/storage.cuh

[2025-10-22T07:54:37.807Z] [INFO]      [exec]             instantiation of "cuco::detail::storage<StorageImpl, T, Extent, Allocator>::storage(Extent, const Allocator &) [with StorageImpl=cuco::storage<1>, T=cudf::size_type, Extent=cuco::valid_extent<int64_t, 18446744073709551615UL>, Allocator=rmm::mr::polymorphic_allocator<char>]" at line 186 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/open_addressing/open_addressing_impl.cuh

[2025-10-22T07:54:37.807Z] [INFO]      [exec]             instantiation of "cuco::detail::open_addressing_impl<Key, Value, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::open_addressing_impl(Extent, double, Value, const KeyEqual &, const ProbingScheme &, const Allocator &, cuda::__4::stream_ref) [with Key=cudf::size_type, Value=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>]" at line 1077 of /opt/rh/gcc-toolset-14/root/usr/include/c++/14/bits/unique_ptr.h

[2025-10-22T07:54:37.807Z] [INFO]      [exec]             instantiation of "std::__detail::__unique_ptr_t<_Tp> std::make_unique<_Tp,_Args...>(_Args &&...) [with _Tp=cuco::detail::open_addressing_impl<cudf::size_type, cudf::size_type, cuco::extent<int64_t, 18446744073709551615UL>, cuda::std::__4::thread_scope_device, cudf::groupby::detail::hash::row_comparator_t, cudf::groupby::detail::hash::probing_scheme_t, rmm::mr::polymorphic_allocator<char>, cuco::storage<1>>, _Args=<cuco::extent<int64_t, 18446744073709551615UL> &, double &, cuco::empty_key<cudf::size_type> &, const cudf::groupby::detail::hash::row_comparator_t &, const cudf::groupby::detail::hash::probing_scheme_t &, const rmm::mr::polymorphic_allocator<char> &, cuda::__4::stream_ref &>]" at line 66 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/static_set/static_set.inl

[2025-10-22T07:54:37.807Z] [INFO]      [exec]             instantiation of "cuco::static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::static_set(Extent, double, cuco::empty_key<Key>, const KeyEqual &, const ProbingScheme &, cuco::cuda_thread_scope<Scope>, Storage, const Allocator &, cuda::__4::stream_ref) [with Key=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>]" at line 121 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:37.807Z] [INFO]      [exec]             instantiation of "std::unique_ptr<cudf::table, std::default_delete<cudf::table>> cudf::groupby::detail::hash::compute_groupby(const cudf::table_view &, cudf::host_span<const cudf::groupby::aggregation_request, 18446744073709551615UL>, __nv_bool, const Equal &, const Hash &, cudf::detail::result_cache *, rmm::cuda_stream_view, rmm::device_async_resource_ref) [with Equal=cudf::groupby::detail::hash::row_comparator_t, Hash=cudf::groupby::detail::hash::row_hash_t]" at line 175 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:37.807Z] [INFO]      [exec] 

[2025-10-22T07:54:37.807Z] [INFO]      [exec] /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/storage/storage_base.cuh(49): error: too few arguments in function call

[2025-10-22T07:54:37.807Z] [INFO]      [exec]     void operator()(pointer ptr) { allocator_.deallocate(ptr, size_); }

[2025-10-22T07:54:37.807Z] [INFO]      [exec]                                                                    ^

[2025-10-22T07:54:37.807Z] [INFO]      [exec]           detected during:

[2025-10-22T07:54:37.807Z] [INFO]      [exec]             instantiation of "void cuco::detail::custom_deleter<SizeType, Allocator>::operator()(cuco::detail::custom_deleter<SizeType, Allocator>::pointer) [with SizeType=int64_t, Allocator=rmm::mr::polymorphic_allocator<cudf::size_type>]" at line 399 of /opt/rh/gcc-toolset-14/root/usr/include/c++/14/bits/unique_ptr.h

[2025-10-22T07:54:37.808Z] [INFO]      [exec]             instantiation of "std::unique_ptr<_Tp, _Dp>::~unique_ptr() noexcept [with _Tp=cudf::size_type, _Dp=cuco::detail::custom_deleter<int64_t, rmm::mr::polymorphic_allocator<cudf::size_type>>]" at line 104 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/storage/bucket_storage.inl

[2025-10-22T07:54:37.808Z] [INFO]      [exec]             instantiation of "cuco::bucket_storage<T, BucketSize, Extent, Allocator>::bucket_storage(Extent, const Allocator &) [with T=cudf::size_type, BucketSize=1, Extent=cuco::valid_extent<int64_t, 18446744073709551615UL>, Allocator=rmm::mr::polymorphic_allocator<char>]" at line 58 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/storage/storage.cuh

[2025-10-22T07:54:37.808Z] [INFO]      [exec]             instantiation of "cuco::detail::storage<StorageImpl, T, Extent, Allocator>::storage(Extent, const Allocator &) [with StorageImpl=cuco::storage<1>, T=cudf::size_type, Extent=cuco::valid_extent<int64_t, 18446744073709551615UL>, Allocator=rmm::mr::polymorphic_allocator<char>]" at line 186 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/open_addressing/open_addressing_impl.cuh

[2025-10-22T07:54:37.808Z] [INFO]      [exec]             instantiation of "cuco::detail::open_addressing_impl<Key, Value, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::open_addressing_impl(Extent, double, Value, const KeyEqual &, const ProbingScheme &, const Allocator &, cuda::__4::stream_ref) [with Key=cudf::size_type, Value=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>]" at line 1077 of /opt/rh/gcc-toolset-14/root/usr/include/c++/14/bits/unique_ptr.h

[2025-10-22T07:54:37.808Z] [INFO]      [exec]             instantiation of "std::__detail::__unique_ptr_t<_Tp> std::make_unique<_Tp,_Args...>(_Args &&...) [with _Tp=cuco::detail::open_addressing_impl<cudf::size_type, cudf::size_type, cuco::extent<int64_t, 18446744073709551615UL>, cuda::std::__4::thread_scope_device, cudf::groupby::detail::hash::row_comparator_t, cudf::groupby::detail::hash::probing_scheme_t, rmm::mr::polymorphic_allocator<char>, cuco::storage<1>>, _Args=<cuco::extent<int64_t, 18446744073709551615UL> &, double &, cuco::empty_key<cudf::size_type> &, const cudf::groupby::detail::hash::row_comparator_t &, const cudf::groupby::detail::hash::probing_scheme_t &, const rmm::mr::polymorphic_allocator<char> &, cuda::__4::stream_ref &>]" at line 66 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/static_set/static_set.inl

[2025-10-22T07:54:37.808Z] [INFO]      [exec]             instantiation of "cuco::static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::static_set(Extent, double, cuco::empty_key<Key>, const KeyEqual &, const ProbingScheme &, cuco::cuda_thread_scope<Scope>, Storage, const Allocator &, cuda::__4::stream_ref) [with Key=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>]" at line 121 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:37.808Z] [INFO]      [exec]             instantiation of "std::unique_ptr<cudf::table, std::default_delete<cudf::table>> cudf::groupby::detail::hash::compute_groupby(const cudf::table_view &, cudf::host_span<const cudf::groupby::aggregation_request, 18446744073709551615UL>, __nv_bool, const Equal &, const Hash &, cudf::detail::result_cache *, rmm::cuda_stream_view, rmm::device_async_resource_ref) [with Equal=cudf::groupby::detail::hash::row_comparator_t, Hash=cudf::groupby::detail::hash::row_hash_t]" at line 175 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:37.808Z] [INFO]      [exec] 

[2025-10-22T07:54:37.808Z] [INFO]      [exec] /opt/rh/gcc-toolset-14/root/usr/include/c++/14/bits/alloc_traits.h(335): error: too few arguments in function call

[2025-10-22T07:54:37.808Z] [INFO]      [exec]         { return __a.allocate(__n); }

[2025-10-22T07:54:37.808Z] [INFO]      [exec]                                  ^

[2025-10-22T07:54:37.808Z] [INFO]      [exec]           detected during:

[2025-10-22T07:54:37.808Z] [INFO]      [exec]             instantiation of "std::allocator_traits<_Alloc>::pointer std::allocator_traits<_Alloc>::allocate(_Alloc &, std::allocator_traits<_Alloc>::size_type) [with _Alloc=rmm::mr::polymorphic_allocator<char>]" at line 846 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/open_addressing/open_addressing_impl.cuh

[2025-10-22T07:54:37.808Z] [INFO]      [exec]             instantiation of "OutputIt cuco::detail::open_addressing_impl<Key, Value, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(OutputIt, cuda::__4::stream_ref) const [with Key=cudf::size_type, Value=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>, OutputIt=cudf::size_type *]" at line 571 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/static_set/static_set.inl

[2025-10-22T07:54:37.808Z] [INFO]      [exec]             instantiation of "OutputIt cuco::static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(OutputIt, cuda::__4::stream_ref) const [with Key=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>, OutputIt=cudf::size_type *]" at line 143 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:37.808Z] [INFO]      [exec]             instantiation of "std::unique_ptr<cudf::table, std::default_delete<cudf::table>> cudf::groupby::detail::hash::compute_groupby(const cudf::table_view &, cudf::host_span<const cudf::groupby::aggregation_request, 18446744073709551615UL>, __nv_bool, const Equal &, const Hash &, cudf::detail::result_cache *, rmm::cuda_stream_view, rmm::device_async_resource_ref) [with Equal=cudf::groupby::detail::hash::row_comparator_t, Hash=cudf::groupby::detail::hash::row_hash_t]" at line 175 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:37.809Z] [INFO]      [exec] 

[2025-10-22T07:54:37.809Z] [INFO]      [exec] /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/open_addressing/open_addressing_impl.cuh(872): error: too few arguments in function call

[2025-10-22T07:54:37.809Z] [INFO]      [exec]         auto d_temp_storage = temp_allocator.allocate(temp_storage_bytes);

[2025-10-22T07:54:37.809Z] [INFO]      [exec]                                                                         ^

[2025-10-22T07:54:37.809Z] [INFO]      [exec]           detected during:

[2025-10-22T07:54:37.809Z] [INFO]      [exec]             instantiation of "OutputIt cuco::detail::open_addressing_impl<Key, Value, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(OutputIt, cuda::__4::stream_ref) const [with Key=cudf::size_type, Value=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>, OutputIt=cudf::size_type *]" at line 571 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/static_set/static_set.inl

[2025-10-22T07:54:37.809Z] [INFO]      [exec]             instantiation of "OutputIt cuco::static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(OutputIt, cuda::__4::stream_ref) const [with Key=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>, OutputIt=cudf::size_type *]" at line 143 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:37.809Z] [INFO]      [exec]             instantiation of "std::unique_ptr<cudf::table, std::default_delete<cudf::table>> cudf::groupby::detail::hash::compute_groupby(const cudf::table_view &, cudf::host_span<const cudf::groupby::aggregation_request, 18446744073709551615UL>, __nv_bool, const Equal &, const Hash &, cudf::detail::result_cache *, rmm::cuda_stream_view, rmm::device_async_resource_ref) [with Equal=cudf::groupby::detail::hash::row_comparator_t, Hash=cudf::groupby::detail::hash::row_hash_t]" at line 175 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:37.809Z] [INFO]      [exec] 

[2025-10-22T07:54:37.809Z] [INFO]      [exec] /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/open_addressing/open_addressing_impl.cuh(892): error: too few arguments in function call

[2025-10-22T07:54:37.809Z] [INFO]      [exec]         temp_allocator.deallocate(d_temp_storage, temp_storage_bytes);

[2025-10-22T07:54:37.809Z] [INFO]      [exec]                                                                     ^

[2025-10-22T07:54:37.809Z] [INFO]      [exec]           detected during:

[2025-10-22T07:54:37.809Z] [INFO]      [exec]             instantiation of "OutputIt cuco::detail::open_addressing_impl<Key, Value, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(OutputIt, cuda::__4::stream_ref) const [with Key=cudf::size_type, Value=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>, OutputIt=cudf::size_type *]" at line 571 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/static_set/static_set.inl

[2025-10-22T07:54:37.809Z] [INFO]      [exec]             instantiation of "OutputIt cuco::static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(OutputIt, cuda::__4::stream_ref) const [with Key=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>, OutputIt=cudf::size_type *]" at line 143 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:37.809Z] [INFO]      [exec]             instantiation of "std::unique_ptr<cudf::table, std::default_delete<cudf::table>> cudf::groupby::detail::hash::compute_groupby(const cudf::table_view &, cudf::host_span<const cudf::groupby::aggregation_request, 18446744073709551615UL>, __nv_bool, const Equal &, const Hash &, cudf::detail::result_cache *, rmm::cuda_stream_view, rmm::device_async_resource_ref) [with Equal=cudf::groupby::detail::hash::row_comparator_t, Hash=cudf::groupby::detail::hash::row_hash_t]" at line 175 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:37.809Z] [INFO]      [exec] 

[2025-10-22T07:54:37.809Z] [INFO]      [exec] /opt/rh/gcc-toolset-14/root/usr/include/c++/14/bits/alloc_traits.h(362): error: too few arguments in function call

[2025-10-22T07:54:37.809Z] [INFO]      [exec]         { __a.deallocate(__p, __n); }

[2025-10-22T07:54:37.809Z] [INFO]      [exec]                                  ^

[2025-10-22T07:54:37.809Z] [INFO]      [exec]           detected during:

[2025-10-22T07:54:37.809Z] [INFO]      [exec]             instantiation of "void std::allocator_traits<_Alloc>::deallocate(_Alloc &, std::allocator_traits<_Alloc>::pointer, std::allocator_traits<_Alloc>::size_type) [with _Alloc=rmm::mr::polymorphic_allocator<char>]" at line 895 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/open_addressing/open_addressing_impl.cuh

[2025-10-22T07:54:37.809Z] [INFO]      [exec]             instantiation of "OutputIt cuco::detail::open_addressing_impl<Key, Value, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(OutputIt, cuda::__4::stream_ref) const [with Key=cudf::size_type, Value=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>, OutputIt=cudf::size_type *]" at line 571 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/static_set/static_set.inl

[2025-10-22T07:54:37.810Z] [INFO]      [exec]             instantiation of "OutputIt cuco::static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(OutputIt, cuda::__4::stream_ref) const [with Key=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>, OutputIt=cudf::size_type *]" at line 143 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:37.810Z] [INFO]      [exec]             instantiation of "std::unique_ptr<cudf::table, std::default_delete<cudf::table>> cudf::groupby::detail::hash::compute_groupby(const cudf::table_view &, cudf::host_span<const cudf::groupby::aggregation_request, 18446744073709551615UL>, __nv_bool, const Equal &, const Hash &, cudf::detail::result_cache *, rmm::cuda_stream_view, rmm::device_async_resource_ref) [with Equal=cudf::groupby::detail::hash::row_comparator_t, Hash=cudf::groupby::detail::hash::row_hash_t]" at line 175 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:37.810Z] [INFO]      [exec] 

[2025-10-22T07:54:38.064Z] [INFO]      [exec] /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/open_addressing/open_addressing_impl.cuh(872): error: too few arguments in function call

[2025-10-22T07:54:38.064Z] [INFO]      [exec]         auto d_temp_storage = temp_allocator.allocate(temp_storage_bytes);

[2025-10-22T07:54:38.064Z] [INFO]      [exec]                                                                         ^

[2025-10-22T07:54:38.064Z] [INFO]      [exec]           detected during:

[2025-10-22T07:54:38.064Z] [INFO]      [exec]             instantiation of "OutputIt cuco::detail::open_addressing_impl<Key, Value, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(OutputIt, cuda::__4::stream_ref) const [with Key=cudf::size_type, Value=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::nullable_row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>, OutputIt=cudf::size_type *]" at line 571 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/static_set/static_set.inl

[2025-10-22T07:54:38.065Z] [INFO]      [exec]             instantiation of "OutputIt cuco::static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(OutputIt, cuda::__4::stream_ref) const [with Key=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::nullable_row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>, OutputIt=cudf::size_type *]" at line 143 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:38.065Z] [INFO]      [exec]             instantiation of "std::unique_ptr<cudf::table, std::default_delete<cudf::table>> cudf::groupby::detail::hash::compute_groupby(const cudf::table_view &, cudf::host_span<const cudf::groupby::aggregation_request, 18446744073709551615UL>, __nv_bool, const Equal &, const Hash &, cudf::detail::result_cache *, rmm::cuda_stream_view, rmm::device_async_resource_ref) [with Equal=cudf::groupby::detail::hash::nullable_row_comparator_t, Hash=cudf::groupby::detail::hash::row_hash_t]" at line 185 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:38.065Z] [INFO]      [exec] 

[2025-10-22T07:54:38.065Z] [INFO]      [exec] /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/open_addressing/open_addressing_impl.cuh(892): error: too few arguments in function call

[2025-10-22T07:54:38.065Z] [INFO]      [exec]         temp_allocator.deallocate(d_temp_storage, temp_storage_bytes);

[2025-10-22T07:54:38.065Z] [INFO]      [exec]                                                                     ^

[2025-10-22T07:54:38.065Z] [INFO]      [exec]           detected during:

[2025-10-22T07:54:38.065Z] [INFO]      [exec]             instantiation of "OutputIt cuco::detail::open_addressing_impl<Key, Value, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(OutputIt, cuda::__4::stream_ref) const [with Key=cudf::size_type, Value=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::nullable_row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>, OutputIt=cudf::size_type *]" at line 571 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cuco-src/include/cuco/detail/static_set/static_set.inl

[2025-10-22T07:54:38.065Z] [INFO]      [exec]             instantiation of "OutputIt cuco::static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(OutputIt, cuda::__4::stream_ref) const [with Key=cudf::size_type, Extent=cuco::extent<int64_t, 18446744073709551615UL>, Scope=cuda::std::__4::thread_scope_device, KeyEqual=cudf::groupby::detail::hash::nullable_row_comparator_t, ProbingScheme=cudf::groupby::detail::hash::probing_scheme_t, Allocator=rmm::mr::polymorphic_allocator<char>, Storage=cuco::storage<1>, OutputIt=cudf::size_type *]" at line 143 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:38.065Z] [INFO]      [exec]             instantiation of "std::unique_ptr<cudf::table, std::default_delete<cudf::table>> cudf::groupby::detail::hash::compute_groupby(const cudf::table_view &, cudf::host_span<const cudf::groupby::aggregation_request, 18446744073709551615UL>, __nv_bool, const Equal &, const Hash &, cudf::detail::result_cache *, rmm::cuda_stream_view, rmm::device_async_resource_ref) [with Equal=cudf::groupby::detail::hash::nullable_row_comparator_t, Hash=cudf::groupby::detail::hash::row_hash_t]" at line 185 of /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu

[2025-10-22T07:54:38.065Z] [INFO]      [exec] 

[2025-10-22T07:54:38.065Z] [INFO]      [exec] 8 errors detected in the compilation of "/home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/_deps/cudf-src/cpp/src/groupby/hash/compute_groupby.cu".

[2025-10-22T07:54:38.319Z] [INFO]      [exec] gmake[2]: *** [_deps/cudf-build/CMakeFiles/cudf.dir/build.make:1338: _deps/cudf-build/CMakeFiles/cudf.dir/src/groupby/hash/compute_groupby.cu.o] Error 1

[2025-10-22T07:54:38.319Z] [INFO]      [exec] gmake[2]: *** Waiting for unfinished jobs....

[2025-10-22T08:05:14.829Z] [INFO]      [exec] gmake[2]: Leaving directory '/home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build'

[2025-10-22T08:05:14.829Z] [INFO]      [exec] gmake[1]: Leaving directory '/home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-458/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build'

[2025-10-22T08:05:14.829Z] [INFO]      [exec] 

[2025-10-22T08:05:14.829Z] [INFO]      [exec] gmake[1]: *** [CMakeFiles/Makefile2:632: _deps/cudf-build/CMakeFiles/cudf.dir/all] Error 2

[2025-10-22T08:05:14.830Z] [INFO]      [exec] gmake: *** [Makefile:156: all] Error 2
```

**Steps/Code to reproduce bug**
Please provide a list of steps or a code sample to reproduce the issue.
Avoid posting private or sensitive data.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Environment details (please complete the following information)**
 - Environment location: [Standalone, YARN, Kubernetes, Cloud(specify cloud provider)]
 - Spark configuration settings related to the issue

--- a/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt
+++ b/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt
@@ -14,14 +14,13 @@
 # limitations under the License.
 #=============================================================================
 
-# Keep the same with https://github.com/rapidsai/rapids-cmake/blob/branch-xx.yy/RAPIDS.cmake
-# the branch-xx.xx in the above line is the branch version
+# Keep the same with https://github.com/rapidsai/rapids-cmake/blob/main/RAPIDS.cmake
 cmake_minimum_required(VERSION 3.30.4 FATAL_ERROR)
 
-# set to the rapids-cmake version
-set(rapids-cmake-version "25.12")
+# set to the rapids-cmake-branch
+set(rapids-cmake-branch "main")
 
-file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-${rapids-cmake-version}/RAPIDS.cmake
+file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/main/RAPIDS.cmake
      ${CMAKE_BINARY_DIR}/RAPIDS.cmake)
 include(${CMAKE_BINARY_DIR}/RAPIDS.cmake)
 

